src/utils/shuffleArray.ts

```diff
@@ -0,0 +1,8 @@
+export const shuffleArray = (array: string[]) => {
+  const newArray = [...array];
```

Comment:
Can you explain to me why you code this?

Reply:
I want the answers to change positions, to avoid the player answering by memorizing the position of the answers.